pkg/services/ngalert/remote/alertmanager.go

```diff
@@ -213,8 +215,21 @@ func (am *Alertmanager) CompareAndSendState(ctx context.Context) error {
 	return nil
 }
 
+// SaveAndApplyConfig should forward the configuration to the remote Alertmanager.
 func (am *Alertmanager) SaveAndApplyConfig(ctx context.Context, cfg *apimodels.PostableUserConfig) error {
-	return nil
+	rawConfig, err := json.Marshal(cfg)
+	if err != nil {
+		return err
+	}
+
+	return am.mimirClient.CreateGrafanaAlertmanagerConfig(
+		ctx,
+		string(rawConfig),
+		fmt.Sprintf("%x", md5.Sum(rawConfig)),
+		1, // TODO: remove
```

Comment:
There's a PR in to remove this field
https://github.com/grafana/grafana/pull/84602
We need the Mimir part to be merged first
https://github.com/grafana/mimir/pull/7643